zipkin/propagation.go

```diff
@@ -28,11 +28,12 @@ func (p Propagator) Inject(
 		return opentracing.ErrInvalidCarrier
 	}
 
-	textMapWriter.Set("x-b3-traceid", strconv.FormatUint(sc.TraceID(), 16))
+	// TODO this needs to change to support 128bit IDs
```

Comment:
@szdavid92 @adriancole FYI. 

BTW the 128bit trace ID is not documented here: http://zipkin.io/pages/instrumenting.html